invenio_rdm_records/records/jsonschemas/records/record-v1.0.0.json

```diff
@@ -77,24 +77,32 @@
       "maximum": 90
     },
 
+    "scheme": {
+      "description": "A scheme.",
+      "type": "string"
+    },
+
+    "identifier": {
+      "description": "An identifier.",
+      "type": "string"
+    },
+
+    "identifiers": {
+      "description": "Identifiers object (keys being scheme, value being the identifier).",
+      "type": "object",
+      "additionalProperties": {"$ref": "#/definitions/identifier"}
+    },
+
     "internal-pid": {
       "type": "object",
       "description": "An internal persistent identifier object.",
       "additionalProperties": false,
-      "required": ["pk", "pid_type", "obj_type", "status"],
+      "required": ["pk", "status"],
       "properties": {
         "pk": {
           "description": "Primary key of the PID object.",
           "type": "integer"
         },
-        "pid_type": {
```

Comment:
Question: Was this fixed in Invenio-Records-Resources as well?

Reply:
Nope only here for now

Reply:
next task is to investigate this